docs/resources/tutorials/cloud_tutorial/ddsrouter.yaml

```diff
@@ -0,0 +1,36 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ddsrouter
+  labels:
+    app: ddsrouter
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ddsrouter
+  template:
+    metadata:
+      labels:
+        app: ddsrouter
+    spec:
+      volumes:
+        - name: config
+          configMap:
+            name: ddsrouter-config
+            items:
+              - key: ddsrouter.config.file
+                path: DDSROUTER_CONFIGURATION.yaml
+      containers:
+        - name: ubuntu-vulcanexus-cloud # TODO(Ensure this docker container exists)
+          image: ubuntu-vulcanexus-cloud:galactic
+          ports:
+            - containerPort: 30001
+              protocol: UDP
+            - containerPort: 30002
+              protocol: UDP
+          volumeMounts:
+            - name: config
+              mountPath: /opt/vulcanexus/galactic/ddsrouter/share/resources
```

Comment:
I would change the mount path to somewhere not dependent on Vulcanexus version. Suggestion: /tmp